src/SharpCode/Structures.cs

```diff
@@ -32,11 +32,12 @@ internal class Property
         public string? Type { get; set; }
         public string? Name { get; set; }
         public Option<string> DefaultValue { get; set; } = Option.None<string>();
+        public Option<string> Documentation { get; set; } = Option.None<string>();
         public Option<string?> Getter { get; set; } = Option.Some<string?>(null);
         public Option<string?> Setter { get; set; } = Option.Some<string?>(null);
     }
 
-    internal class Parameter
+    internal class Parameter 
```

Comment:
Can you please remove trailing whitespace 😁 FxCop Analyzers coming soon to take care of that, but not there yet :(